src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-purge-modal/rbd-trash-purge-modal.component.html

```diff
@@ -10,7 +10,7 @@
           novalidate>
       <div class="modal-body">
         <p>
-          <ng-container i18n>To purge, select one or All images and click</ng-container>&nbsp;
+          <ng-container i18n>To purge, select one or All pools and click</ng-container>&nbsp;
```

Comment:
What about using
```suggestion
          <ng-container i18n>To purge, select one or more pools and click</ng-container>&nbsp;
```

Reply:
In the purge trash window we have option to select All or one pool of our choice, so I think "more pools" will not be suitable. Can we discuss this on today's standup?